flexmeasures/ui/templates/crud/users.html

```diff
@@ -7,72 +7,74 @@
-                        {% for user in users %}
-                        <tr>
-                            <td>
-                                {{ user.username }}
-                            </td>
-                            <td>
-                                <a href="mailto:{{ user.email }}" title="Mail this user">{{ user.email }}</a>
-                            </td>
-                                    <td>{% for role in user.flexmeasures_roles %}
-                                          {{ role.name }}{{ "," if not loop.last }}
-                                        {% endfor %}
-                            </td>
-                            <td>
-                                <a href="/accounts/{{user.account.id}}" title="View accounts">{{ user.account.name }}</a>
-                            </td>
-                            <td>
-                                {{ user.timezone }}
-                            </td>
-                            <td title="{{  user.last_login_at | localized_datetime }}" data-sort="{{ user.last_login_at }}">
-                                {{ user.last_login_at | naturalized_datetime}}
-                            </td>
-                            <td title="{{  user.last_seen_at | localized_datetime }}" data-sort="{{ user.last_seen_at }}">
-                                {{ user.last_seen_at | naturalized_datetime}}
-                            </td>
-                            <td>
-                                {{ user.active }}
-                            </td>
-                            <td class="hidden">
-                                /users/{{ user.id }}
-                            </td>
-                        </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-        </div>
-        <div class="col-sm-2"></div>
+  <div class="row">
+    <div class="col-md-2"></div>
+    <div class="col-md-8">
+      <div class="card">
+        <h3>All {% if not include_inactive %}active{% endif %}users</h3>
+        <form class="form-inline float-right" role="form" target="" method="get" id="user-list-options">
+          <div class="form-check form-check-inline">
+            <label class="form-check-label">
+              <input name="include_inactive" {% if include_inactive %} checked="checked"{% endif %} type="checkbox"> Include inactive
+            </label>
+          </div>
+        </form>
+        <div class="table-responsive">
+        <table class="table table-striped table-responsive paginate nav-on-click" title="View/edit this user">
+          <thead>
+            <tr>
+              <th>Username</th>
+              <th>Email</th>
+              <th>Roles</th>
+              <th>Account</th>
+              <th>Timezone</th>
+              <th>Last Login</th>
+              <th>Last Seen</th>
+              <th>Active</th>
+              <th class="d-none" style="display:none;">URL</th>
```

Comment:
More `style="display:none;` that can go. (I won't comment on all)

Reply:
i've sort that one out

Reply:
removed the custom css